class1.html

```diff
@@ -183,29 +197,29 @@
               </li>
               <li>
                   <a href="http://www.python.org/download/" target="_blank">Install Python 3</a>
-                  <br/><small>(This step is for Windows users only. GNU/Linux and Mac OS X come with Python installed!)</small>
               </li>
               <li>
                   <strong>Windows Users:</strong> After installing Python, open the "powershell" program and type:
-                  <br/><code class="copy--xsmall">[Environment]::SetEnvironmentVariable("Path", "$env:Path;C:\Python34", "User")</code>
+                  <br/><code class="copy--xsmall">[Environment]::SetEnvironmentVariable("Path", "$env:Path;C:\Python35", "User")</code>
               </li>
               <li>
-                  Locate and run the terminal program. Type <code>python</code> and hit enter.
+                  Locate and run the terminal program. Type <code>python3</code> and hit enter.
```

Comment:
switched out all the executable names. This is at least correct for osx and may be correct for windows.